### PR TITLE
Add O365DefenderSafeLinks  atp-policy-get and  atp-policy-set commands

### PR DESCRIPTION
**Pull request by thefrieddan1**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-hq.paloaltonetworks.local/browse/CIAC-2173

## Description
Added two commands to O365DefenderSafeLinks integration. 
atp-policy-get and atp-policy-set .

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

### CODE REVIEW

File: Packs/Microsoft365Defender/Integrations/O365DefenderSafeLinks/O365DefenderSafeLinks.ps1 [hunk 1]
```diff
@@ -484,6 +484,56 @@ class ExchangeOnlinePowershellV2Client {
         https://docs.microsoft.com/en-us/powershell/module/exchange/get-safelinksaggregatereport?view=exchange-ps
         #>
     }
+    [PSObject]
+    GetAtpPolicy() {
+        $this.CreateSession()
+        $results = Get-AtpPolicyForO365
+        return $results
+        $this.DisconnectSession()
```
Comment (DeanArbel) [inline Packs/Microsoft365Defender/Integrations/O365DefenderSafeLinks/O365DefenderSafeLinks.ps1:492]: Please put this command in a try/finally block like other commands.

File: Packs/Microsoft365Defender/Integrations/O365DefenderSafeLinks/O365DefenderSafeLinks.ps1 [hunk 2]
```diff
@@ -615,6 +665,38 @@ function GetAggregateReportCommand {
...
+    [OutputType([System.Object[]])]
+    Param (
+        [Parameter(Mandatory)][ExchangeOnlinePowershellV2Client]$client,
+        [hashtable]$kwargs
+    )
+
+    $raw_response = $client.SetAtpPolicy($kwargs)
+    # $human_readable = TableToMarkdown $raw_response "Results of $command"
+    $entry_context = @{ "$script:INTEGRATION_ENTRY_CONTEXT.AtpPolicy" = CreateContextForReport $raw_response }
+    $human_readable = "OK"
```
Comment (DeanArbel) [inline Packs/Microsoft365Defender/Integrations/O365DefenderSafeLinks/O365DefenderSafeLinks.ps1:696]: ?

File: Packs/Microsoft365Defender/ReleaseNotes/4_4_14.md [hunk 3]
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### O365 Defender SafeLinks
+- Added support for ***atp-policy-get*** and ***atp-policy-set*** commands.
```
Comment (DeanArbel) [inline Packs/Microsoft365Defender/ReleaseNotes/4_4_14.md:4]:
```suggestion
- Added new commands ***atp-policy-get*** and ***atp-policy-set***.
```

File: Packs/Microsoft365Defender/pack_metadata.json [hunk 4]
```diff
@@ -2,7 +2,7 @@
     "name": "Microsoft 365 Defender",
     "description": "Microsoft 365 Defender is a unified pre- and post-breach enterprise defense suite that natively coordinates detection, prevention, investigation, and response across endpoints, identities, email, and applications to provide integrated protection against sophisticated attacks.",
     "support": "xsoar",
-    "currentVersion": "4.4.13",
+    "currentVersion": "4.4.14",
```
Comment (DeanArbel) [inline Packs/Microsoft365Defender/pack_metadata.json:5]: This can be a minor version.

**Review by DeanArbel — CHANGES_REQUESTED** [review]

**Review by DeanArbel — APPROVED** [review]
Looks great

File: Packs/Microsoft365Defender/Integrations/O365DefenderSafeLinks/O365DefenderSafeLinks.yml [hunk 5]
```diff
@@ -1013,6 +1013,65 @@ script:
     - contextPath: O365Defender.SafeLinks.AggregateReport.ReportId
       description: "The report id, unique for every run"
       type: Number
+  - name: o365-defender-safelinks-atp-policy-get
+    description: Get APT policy. In order to manage BlockURLs please use the command o365-defender-safelinks-global-url-blocklist.
```
Comment (ShirleyDenkberg) [inline Packs/Microsoft365Defender/Integrations/O365DefenderSafeLinks/O365DefenderSafeLinks.yml:1017]:
```suggestion
    description: Get APT policy. In order to manage BlockURLs, use the o365-defender-safelinks-global-url-blocklist  command.
```

File: Packs/Microsoft365Defender/Integrations/O365DefenderSafeLinks/O365DefenderSafeLinks.yml [hunk 6]
```diff
@@ -1013,6 +1013,65 @@ script:
...
+    deprecated: false
+    outputs:
+    - contextPath: O365Defender.SafeLinks.AtpPolicy.Name
+      description: ATP policy name.
+      type: String
+    - contextPath: O365Defender.SafeLinks.AtpPolicy.AdminDisplayName
+      description: ATP policy admin display name.
+      type: String
+    - contextPath: O365Defender.SafeLinks.AtpPolicy.EnableATPForSPOTeamsODB
+      description: ATP policy enable for SPOT teams.
```
Comment (ShirleyDenkberg) [inline Packs/Microsoft365Defender/Integrations/O365DefenderSafeLinks/O365DefenderSafeLinks.yml:1027]:
```suggestion
      description: ATP policy enabled for SPOT teams.
```

File: Packs/Microsoft365Defender/Integrations/O365DefenderSafeLinks/O365DefenderSafeLinks.yml [hunk 7]
```diff
@@ -1013,6 +1013,65 @@ script:
...
+      type: String
+    - contextPath: O365Defender.SafeLinks.AtpPolicy.AllowSafeDocsOpen
+      description: ATP policy allow safe docs open.
```
Comment (ShirleyDenkberg) [inline Packs/Microsoft365Defender/Integrations/O365DefenderSafeLinks/O365DefenderSafeLinks.yml:1030]: Should this be:
Whether the ATP policy allows safe docs to open.

File: Packs/Microsoft365Defender/Integrations/O365DefenderSafeLinks/O365DefenderSafeLinks.yml [hunk 8]
```diff
@@ -1013,6 +1013,65 @@ script:
...
+      type: String
+    - contextPath: O365Defender.SafeLinks.AtpPolicy.EnableSafeDocs
+      description: ATP policy enable safe docs.
```
Comment (ShirleyDenkberg) [inline Packs/Microsoft365Defender/Integrations/O365DefenderSafeLinks/O365DefenderSafeLinks.yml:1033]: Should this be:
Whether the ATP policy enables safe docs.

File: Packs/Microsoft365Defender/Integrations/O365DefenderSafeLinks/O365DefenderSafeLinks.yml [hunk 9]
```diff
@@ -1013,6 +1013,65 @@ script:
...
+      type: String
+    - contextPath: O365Defender.SafeLinks.AtpPolicy.Identity
+      description: ATP policy id.
```
Comment (ShirleyDenkberg) [inline Packs/Microsoft365Defender/Integrations/O365DefenderSafeLinks/O365DefenderSafeLinks.yml:1036]:
```suggestion
      description: ATP policy ID.
```

File: Packs/Microsoft365Defender/Integrations/O365DefenderSafeLinks/O365DefenderSafeLinks.yml [hunk 10]
```diff
@@ -1013,6 +1013,65 @@ script:
...
+      type: String
+    - contextPath: O365Defender.SafeLinks.AtpPolicy.IsValid
+      description: Is ATP policy valid.
```
Comment (ShirleyDenkberg) [inline Packs/Microsoft365Defender/Integrations/O365DefenderSafeLinks/O365DefenderSafeLinks.yml:1039]:
```suggestion
      description: Is the ATP policy valid.
```

File: Packs/Microsoft365Defender/Integrations/O365DefenderSafeLinks/O365DefenderSafeLinks.yml [hunk 11]
```diff
@@ -1013,6 +1013,65 @@ script:
...
+      type: String
+    - contextPath: O365Defender.SafeLinks.AtpPolicy.WhenCreatedUTC
+      description: When ATP policy created.
```
Comment (ShirleyDenkberg) [inline Packs/Microsoft365Defender/Integrations/O365DefenderSafeLinks/O365DefenderSafeLinks.yml:1042]:
```suggestion
      description: When the ATP policy was created in UTC format.
```

File: Packs/Microsoft365Defender/Integrations/O365DefenderSafeLinks/O365DefenderSafeLinks.yml [hunk 12]
```diff
@@ -1013,6 +1013,65 @@ script:
...
+      type: String
+    - contextPath: O365Defender.SafeLinks.AtpPolicy.WhenChangedUTC
+      description: When ATP policy changed.
```
Comment (ShirleyDenkberg) [inline Packs/Microsoft365Defender/Integrations/O365DefenderSafeLinks/O365DefenderSafeLinks.yml:1045]:
```suggestion
      description: When the ATP policy was changed in UTC format.
```

File: Packs/Microsoft365Defender/Integrations/O365DefenderSafeLinks/O365DefenderSafeLinks.yml [hunk 13]
```diff
@@ -1013,6 +1013,65 @@ script:
...
+      type: String
+  - name: o365-defender-safelinks-atp-policy-set
+    description: Set ATP policy.
```
Comment (ShirleyDenkberg) [inline Packs/Microsoft365Defender/Integrations/O365DefenderSafeLinks/O365DefenderSafeLinks.yml:1048]:
```suggestion
    description: Set the ATP policy.
```

File: Packs/Microsoft365Defender/Integrations/O365DefenderSafeLinks/O365DefenderSafeLinks.yml [hunk 14]
```diff
@@ -1013,6 +1013,65 @@ script:
...
+    deprecated: false
+    arguments:
+    - name: allow_safe_docs_open
+      description: Toggle mode for allow open safe documents.
```
Comment (ShirleyDenkberg) [inline Packs/Microsoft365Defender/Integrations/O365DefenderSafeLinks/O365DefenderSafeLinks.yml:1052]:
```suggestion
      description: Toggle mode to allow opening safe documents.
```

File: Packs/Microsoft365Defender/Integrations/O365DefenderSafeLinks/O365DefenderSafeLinks.yml [hunk 15]
```diff
@@ -1013,6 +1013,65 @@ script:
...
+      required: false
+      isArray: false
+      auto: PREDEFINED
+      predefined:
+      - "true"
+      - "false"
+    - name: enable_atp_spo_teams_odb
+      description: Toggle mode for enable_atp_spo_teams_odb.
```
Comment (ShirleyDenkberg) [inline Packs/Microsoft365Defender/Integrations/O365DefenderSafeLinks/O365DefenderSafeLinks.yml:1060]: Need a better description. I don't know what enable_atp_spo_teams_odb means.

File: Packs/Microsoft365Defender/Integrations/O365DefenderSafeLinks/O365DefenderSafeLinks.yml [hunk 16]
```diff
@@ -1013,6 +1013,65 @@ script:
...
+      required: false
+      isArray: false
+      auto: PREDEFINED
+      predefined:
+      - "true"
+      - "false"
+    - name: enable_safe_docs
+      description: Toggle mode for enable_safe_docs.
```
Comment (ShirleyDenkberg) [inline Packs/Microsoft365Defender/Integrations/O365DefenderSafeLinks/O365DefenderSafeLinks.yml:1068]:
```suggestion
      description: Toggle mode to enable safe docs.
```

File: Packs/Microsoft365Defender/Integrations/O365DefenderSafeLinks/README.md [hunk 17]
```diff
@@ -1087,3 +1087,49 @@ Known Limitations
 * Allow up to 30 minutes for a new or updated policy to be applied.
 * Organization - be sure to use an `.onmicrosoft.com` domain in the Organization parameter value. Otherwise, you might encounter cryptic permission issues when you run commands in the app context.
 
+### o365-defender-safelinks-atp-policy-get
+***
+Get APT policy. In order to manage BlockURLs please use the command o365-defender-safelinks-global-url-blocklist.
```
Comment (ShirleyDenkberg) [inline Packs/Microsoft365Defender/Integrations/O365DefenderSafeLinks/README.md:1092]:
```suggestion
Get the APT policy. In order to manage BlockURLs, use the o365-defender-safelinks-global-url-blocklist command .
```

File: Packs/Microsoft365Defender/Integrations/O365DefenderSafeLinks/README.md [hunk 18]
```diff
@@ -1087,3 +1087,49 @@ Known Limitations
...
+There are no input arguments for this command.
+
+#### Context Output
+
+| **Path** | **Type** | **Description** |
+| --- | --- | --- |
+| O365Defender.SafeLinks.AtpPolicy.Name | String | ATP policy name. | 
+| O365Defender.SafeLinks.AtpPolicy.AdminDisplayName | String | ATP policy admin display name. | 
+| O365Defender.SafeLinks.AtpPolicy.EnableATPForSPOTeamsODB | String | ATP policy enable for SPOT teams. | 
+| O365Defender.SafeLinks.AtpPolicy.AllowSafeDocsOpen | String | ATP policy allow safe docs open. |
```
Comment (ShirleyDenkberg) [inline Packs/Microsoft365Defender/Integrations/O365DefenderSafeLinks/README.md:1109]:
```suggestion
| O365Defender.SafeLinks.AtpPolicy.AllowSafeDocsOpen | String | Whether the ATP policy allows safe docs open. | 
```

File: Packs/Microsoft365Defender/Integrations/O365DefenderSafeLinks/README.md [hunk 19]
```diff
@@ -1087,3 +1087,49 @@ Known Limitations
...
+
+#### Context Output
+
+| **Path** | **Type** | **Description** |
+| --- | --- | --- |
+| O365Defender.SafeLinks.AtpPolicy.Name | String | ATP policy name. | 
+| O365Defender.SafeLinks.AtpPolicy.AdminDisplayName | String | ATP policy admin display name. | 
+| O365Defender.SafeLinks.AtpPolicy.EnableATPForSPOTeamsODB | String | ATP policy enable for SPOT teams. | 
+| O365Defender.SafeLinks.AtpPolicy.AllowSafeDocsOpen | String | ATP policy allow safe docs open. | 
+| O365Defender.SafeLinks.AtpPolicy.EnableSafeDocs | String | ATP policy enable safe docs. |
```
Comment (ShirleyDenkberg) [inline Packs/Microsoft365Defender/Integrations/O365DefenderSafeLinks/README.md:1110]:
```suggestion
| O365Defender.SafeLinks.AtpPolicy.EnableSafeDocs | String | Whether the ATP policy enables safe docs. | 
```

File: Packs/Microsoft365Defender/Integrations/O365DefenderSafeLinks/README.md [hunk 20]
```diff
@@ -1087,3 +1087,49 @@ Known Limitations
...
+#### Context Output
+
+| **Path** | **Type** | **Description** |
+| --- | --- | --- |
+| O365Defender.SafeLinks.AtpPolicy.Name | String | ATP policy name. | 
+| O365Defender.SafeLinks.AtpPolicy.AdminDisplayName | String | ATP policy admin display name. | 
+| O365Defender.SafeLinks.AtpPolicy.EnableATPForSPOTeamsODB | String | ATP policy enable for SPOT teams. | 
+| O365Defender.SafeLinks.AtpPolicy.AllowSafeDocsOpen | String | ATP policy allow safe docs open. | 
+| O365Defender.SafeLinks.AtpPolicy.EnableSafeDocs | String | ATP policy enable safe docs. | 
+| O365Defender.SafeLinks.AtpPolicy.Identity | String | ATP policy id. |
```
Comment (ShirleyDenkberg) [inline Packs/Microsoft365Defender/Integrations/O365DefenderSafeLinks/README.md:1111]:
```suggestion
| O365Defender.SafeLinks.AtpPolicy.Identity | String | ATP policy ID. | 
```

File: Packs/Microsoft365Defender/Integrations/O365DefenderSafeLinks/README.md [hunk 21]
```diff
@@ -1087,3 +1087,49 @@ Known Limitations
...
+
+| **Path** | **Type** | **Description** |
+| --- | --- | --- |
+| O365Defender.SafeLinks.AtpPolicy.Name | String | ATP policy name. | 
+| O365Defender.SafeLinks.AtpPolicy.AdminDisplayName | String | ATP policy admin display name. | 
+| O365Defender.SafeLinks.AtpPolicy.EnableATPForSPOTeamsODB | String | ATP policy enable for SPOT teams. | 
+| O365Defender.SafeLinks.AtpPolicy.AllowSafeDocsOpen | String | ATP policy allow safe docs open. | 
+| O365Defender.SafeLinks.AtpPolicy.EnableSafeDocs | String | ATP policy enable safe docs. | 
+| O365Defender.SafeLinks.AtpPolicy.Identity | String | ATP policy id. | 
+| O365Defender.SafeLinks.AtpPolicy.IsValid | String | Is ATP policy valid. |
```
Comment (ShirleyDenkberg) [inline Packs/Microsoft365Defender/Integrations/O365DefenderSafeLinks/README.md:1112]:
```suggestion
| O365Defender.SafeLinks.AtpPolicy.IsValid | String | Is the ATP policy valid. | 
```

File: Packs/Microsoft365Defender/Integrations/O365DefenderSafeLinks/README.md [hunk 22]
```diff
@@ -1087,3 +1087,49 @@ Known Limitations
...
+
+There are no input arguments for this command.
+
+#### Context Output
+
+| **Path** | **Type** | **Description** |
+| --- | --- | --- |
+| O365Defender.SafeLinks.AtpPolicy.Name | String | ATP policy name. | 
+| O365Defender.SafeLinks.AtpPolicy.AdminDisplayName | String | ATP policy admin display name. | 
+| O365Defender.SafeLinks.AtpPolicy.EnableATPForSPOTeamsODB | String | ATP policy enable for SPOT teams. |
```
Comment (ShirleyDenkberg) [inline Packs/Microsoft365Defender/Integrations/O365DefenderSafeLinks/README.md:1108]: Should this be:
Whether the ATP policy was enabled for SPOT teams.

File: Packs/Microsoft365Defender/Integrations/O365DefenderSafeLinks/README.md [hunk 23]
```diff
@@ -1087,3 +1087,49 @@ Known Limitations
...
+| **Path** | **Type** | **Description** |
+| --- | --- | --- |
+| O365Defender.SafeLinks.AtpPolicy.Name | String | ATP policy name. | 
+| O365Defender.SafeLinks.AtpPolicy.AdminDisplayName | String | ATP policy admin display name. | 
+| O365Defender.SafeLinks.AtpPolicy.EnableATPForSPOTeamsODB | String | ATP policy enable for SPOT teams. | 
+| O365Defender.SafeLinks.AtpPolicy.AllowSafeDocsOpen | String | ATP policy allow safe docs open. | 
+| O365Defender.SafeLinks.AtpPolicy.EnableSafeDocs | String | ATP policy enable safe docs. | 
+| O365Defender.SafeLinks.AtpPolicy.Identity | String | ATP policy id. | 
+| O365Defender.SafeLinks.AtpPolicy.IsValid | String | Is ATP policy valid. | 
+| O365Defender.SafeLinks.AtpPolicy.WhenCreatedUTC | String | When ATP policy created. |
```
Comment (ShirleyDenkberg) [inline Packs/Microsoft365Defender/Integrations/O365DefenderSafeLinks/README.md:1113]:
```suggestion
| O365Defender.SafeLinks.AtpPolicy.WhenCreatedUTC | String | When the ATP policy was created in UTC format. | 
```

File: Packs/Microsoft365Defender/Integrations/O365DefenderSafeLinks/README.md [hunk 24]
```diff
@@ -1087,3 +1087,49 @@ Known Limitations
...
+| --- | --- | --- |
+| O365Defender.SafeLinks.AtpPolicy.Name | String | ATP policy name. | 
+| O365Defender.SafeLinks.AtpPolicy.AdminDisplayName | String | ATP policy admin display name. | 
+| O365Defender.SafeLinks.AtpPolicy.EnableATPForSPOTeamsODB | String | ATP policy enable for SPOT teams. | 
+| O365Defender.SafeLinks.AtpPolicy.AllowSafeDocsOpen | String | ATP policy allow safe docs open. | 
+| O365Defender.SafeLinks.AtpPolicy.EnableSafeDocs | String | ATP policy enable safe docs. | 
+| O365Defender.SafeLinks.AtpPolicy.Identity | String | ATP policy id. | 
+| O365Defender.SafeLinks.AtpPolicy.IsValid | String | Is ATP policy valid. | 
+| O365Defender.SafeLinks.AtpPolicy.WhenCreatedUTC | String | When ATP policy created. | 
+| O365Defender.SafeLinks.AtpPolicy.WhenChangedUTC | String | When ATP policy changed. |
```
Comment (ShirleyDenkberg) [inline Packs/Microsoft365Defender/Integrations/O365DefenderSafeLinks/README.md:1114]:
```suggestion
| O365Defender.SafeLinks.AtpPolicy.WhenChangedUTC | String | When the ATP policy was changed in UTC format. | 
```

File: Packs/Microsoft365Defender/Integrations/O365DefenderSafeLinks/README.md [hunk 25]
```diff
@@ -1087,3 +1087,49 @@ Known Limitations
...
+| O365Defender.SafeLinks.AtpPolicy.AllowSafeDocsOpen | String | ATP policy allow safe docs open. | 
+| O365Defender.SafeLinks.AtpPolicy.EnableSafeDocs | String | ATP policy enable safe docs. | 
+| O365Defender.SafeLinks.AtpPolicy.Identity | String | ATP policy id. | 
+| O365Defender.SafeLinks.AtpPolicy.IsValid | String | Is ATP policy valid. | 
+| O365Defender.SafeLinks.AtpPolicy.WhenCreatedUTC | String | When ATP policy created. | 
+| O365Defender.SafeLinks.AtpPolicy.WhenChangedUTC | String | When ATP policy changed. | 
+
+### o365-defender-safelinks-atp-policy-set
+***
+Set ATP policy.
```
Comment (ShirleyDenkberg) [inline Packs/Microsoft365Defender/Integrations/O365DefenderSafeLinks/README.md:1118]:
```suggestion
Set the ATP policy.
```

File: Packs/Microsoft365Defender/Integrations/O365DefenderSafeLinks/README.md [hunk 26]
```diff
@@ -1087,3 +1087,49 @@ Known Limitations
...
+
+
+#### Base Command
+
+`o365-defender-safelinks-atp-policy-set`
+#### Input
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+| allow_safe_docs_open | Toggle mode for allow open safe documents. Possible values are: true, false. | Optional |
```
Comment (ShirleyDenkberg) [inline Packs/Microsoft365Defender/Integrations/O365DefenderSafeLinks/README.md:1128]:
```suggestion
| allow_safe_docs_open | Whether to allow opening safe documents. Possible values are: true, false. | Optional | 
```

File: Packs/Microsoft365Defender/Integrations/O365DefenderSafeLinks/README.md [hunk 27]
```diff
@@ -1087,3 +1087,49 @@ Known Limitations
...
+
+#### Base Command
+
+`o365-defender-safelinks-atp-policy-set`
+#### Input
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+| allow_safe_docs_open | Toggle mode for allow open safe documents. Possible values are: true, false. | Optional | 
+| enable_atp_spo_teams_odb | Toggle mode for enable_atp_spo_teams_odb. Possible values are: true, false. | Optional |
```
Comment (ShirleyDenkberg) [inline Packs/Microsoft365Defender/Integrations/O365DefenderSafeLinks/README.md:1129]:
```suggestion
| enable_atp_spo_teams_odb | Whether the ATP policy was enabled for SPOT teams. Possible values are: true, false. | Optional | 
```

File: Packs/Microsoft365Defender/Integrations/O365DefenderSafeLinks/README.md [hunk 28]
```diff
@@ -1087,3 +1087,49 @@ Known Limitations
...
+#### Base Command
+
+`o365-defender-safelinks-atp-policy-set`
+#### Input
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+| allow_safe_docs_open | Toggle mode for allow open safe documents. Possible values are: true, false. | Optional | 
+| enable_atp_spo_teams_odb | Toggle mode for enable_atp_spo_teams_odb. Possible values are: true, false. | Optional | 
+| enable_safe_docs | Toggle mode for enable_safe_docs. Possible values are: true, false. | Optional |
```
Comment (ShirleyDenkberg) [inline Packs/Microsoft365Defender/Integrations/O365DefenderSafeLinks/README.md:1130]:
```suggestion
| enable_safe_docs | Whether to enable safe docs. Possible values are: true, false. | Optional | 
```

File: Packs/Microsoft365Defender/ReleaseNotes/4_4_14.md [hunk 29]
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### O365 Defender SafeLinks
+- Added new commands ***atp-policy-get*** and ***atp-policy-set***.
```
Comment (ShirleyDenkberg) [inline Packs/Microsoft365Defender/ReleaseNotes/4_4_14.md:4]: Added the following commands:
- ***atp-policy-get***
- ***atp-policy-set***

**Comment by ShirleyDenkberg** [comment]
@DeanArbel Doc review completed.

**Pull request merged**